src/get-started/wordcount-example.md

```diff
@@ -677,15 +801,26 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WindowedWordCo
      -Pdataflow-runner
 ```
 
-To view the full code in Java, see **[WindowedWordCount](https://github.com/apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java).**
+To view the full code in Java, see
+**[WindowedWordCount](https://github.com/apache/beam/blob/master/examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java).**
 
 > **Note:** WindowedWordCount is not yet available for the Python SDK.
 
 ### Unbounded and bounded pipeline input modes
 
-Beam allows you to create a single pipeline that can handle both bounded and unbounded types of input. If the input is unbounded, then all PCollections of the pipeline will be unbounded as well. The same goes for bounded input. If your input has a fixed number of elements, it's considered a 'bounded' data set. If your input is continuously updating, then it's considered 'unbounded'.
+Beam allows you to create a single pipeline that can handle both bounded and
+unbounded types of input. If your input has a fixed number of elements, it's
+considered a 'bounded' data set. If your input is continuously updating, then
```

Comment:
I'd file a Jira to update the wording here - the way I think of unbounded is that it doesn't make sense to ask "do I have all of the data", just "at what point do I have all of the data up to", but that might need to be workshopped a little.